tests/e2e/suite_test.go

```diff
@@ -124,9 +136,9 @@ var _ = AfterSuite(func() {
 	cancel()
 	By("tearing down test environment")
 	close(stopCh)
-	if disableTestArtifacts != "true" {
-		getPodLogs(kubeClient, Namespace, NumaplaneLabel, "manager", filepath.Join(ControllerOutputPath, "numaplane-controller.log"))
-	}
+	// if disableTestArtifacts != "true" {
+	// 	getPodLogs(kubeClient, Namespace, NumaplaneLabel, "manager", filepath.Join(ControllerOutputPath, "numaplane-controller.log"))
+	// }
```

Comment:
should we clean this up or are you leaving it there just in case we need it?

Reply:
Should be cleaned up now.